test/functional/feature_block.py

```diff
@@ -85,12 +85,12 @@ def set_test_params(self):
         # Very large reorgs cause cs_main to be held for a very long time in ActivateBestChainStep,
         # which causes RPC to hang, so we need to increase RPC timeouts
         self.rpc_timeout = 180
-        # Must set '-dip3params=2000:2000' to create pre-dip3 blocks only
         self.extra_args = [[
-            '-dip3params=2000:2000',
+            '-dip3params=2000:2000', # Must set '-dip3params=2000:2000' to create pre-dip3 blocks only
```

Comment:
why not use testactivationheight?

Reply:
it's out of scope this PR, I will do that in PR where I set DIP3 activation from block 1